src/utils/constants.js

```diff
@@ -1,132 +0,0 @@
-    link: "#",
-  },
-  {
-    alt: "a dog",
-    src: "https://i.imgflip.com/5s7g8s.png",
-    keyword: "Nature",
-    title: "Everyone Needs a Special 'Sit Spot' in Nature",
-    text: `Ever since I read Richard Louv's influential book, "Last Child in the Woods," the idea of having a special "sit spot" has stuck with me. This advice, which Louv attributes to nature educator Jon Young, is for both adults and children to find...`,
-    date: "November 4, 2020",
-    source: "treehugger",
-    link: "#",
-  },
-  {
-    alt: "a dog",
-    src: "https://i.imgflip.com/5s7g8s.png",
-    keyword: "Nature",
-    title: "Everyone Needs a Special 'Sit Spot' in Nature",
-    text: `Ever since I read Richard Louv's influential book, "Last Child in the Woods," the idea of having a special "sit spot" has stuck with me. This advice, which Louv attributes to nature educator Jon Young, is for both adults and children to find...`,
-    date: "November 4, 2020",
-    source: "treehugger",
-    link: "#",
-  },
-  {
-    alt: "a dog",
-    src: "https://i.imgflip.com/5s7g8s.png",
-    keyword: "Nature",
-    title: "Everyone Needs a Special 'Sit Spot' in Nature",
-    text: `Ever since I read Richard Louv's influential book, "Last Child in the Woods," the idea of having a special "sit spot" has stuck with me. This advice, which Louv attributes to nature educator Jon Young, is for both adults and children to find...`,
-    date: "November 4, 2020",
-    source: "treehugger",
-    link: "#",
-  },
-  {
-    alt: "a dog",
-    src: "https://i.imgflip.com/5s7g8s.png",
-    keyword: "Nature",
-    title: "Everyone Needs a Special 'Sit Spot' in Nature",
-    text: `Ever since I read Richard Louv's influential book, "Last Child in the Woods," the idea of having a special "sit spot" has stuck with me. This advice, which Louv attributes to nature educator Jon Young, is for both adults and children to find...`,
-    date: "November 4, 2020",
-    source: "treehugger",
-    link: "#",
-  },
-  {
-    alt: "a dog",
-    src: "https://i.imgflip.com/5s7g8s.png",
-    keyword: "Nature",
-    title: "Everyone Needs a Special 'Sit Spot' in Nature",
-    text: `Ever since I read Richard Louv's influential book, "Last Child in the Woods," the idea of having a special "sit spot" has stuck with me. This advice, which Louv attributes to nature educator Jon Young, is for both adults and children to find...`,
-    date: "November 4, 2020",
-    source: "treehugger",
-    link: "#",
-  },
-  {
-    alt: "a dog",
-    src: "https://i.imgflip.com/5s7g8s.png",
-    keyword: "Nature",
-    title: "Everyone Needs a Special 'Sit Spot' in Nature",
-    text: `Ever since I read Richard Louv's influential book, "Last Child in the Woods," the idea of having a special "sit spot" has stuck with me. This advice, which Louv attributes to nature educator Jon Young, is for both adults and children to find...`,
-    date: "November 4, 2020",
-    source: "treehugger",
-    link: "#",
-  },
-  {
```

Comment:
`  const [pageTheme, setpageTheme] = React.useState(false);`


`setpageTheme` should be in `camelCase`


```js

setPageTheme

```